### PR TITLE
Code early stages

### PR DESCRIPTION
**Pull request by Trey50Daniel**
Updated UI + created main menu with keyboard functionality

### CODE REVIEW

File: Assets/Scripts/LevelManager.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.SceneManagement;
+
+public class LevelManager : MonoBehaviour {
```
Comment (kinbiko) [inline Assets/Scripts/LevelManager.cs]: Use `LevelHandler` instead, this file should be deleted

File: Assets/Scripts/MainMenuManager.cs [hunk 2]
```diff
@@ -0,0 +1,80 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.SceneManagement;
+
+public class MainMenuManager : MonoBehaviour {
+
+    private string[] menuItems = {"Start", "Exit"};
```
Comment (kinbiko) [inline Assets/Scripts/MainMenuManager.cs]: Exit doesn't make sense on the web. We're making an About page soon, so change to that.

File: Assets/Scripts/MainMenuManager.cs [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    private int selectedIndex;
+
+    void Start()
+    {
+        selectedIndex = 0;
+    }
+
+    public void Exit()
```
Comment (kinbiko) [inline Assets/Scripts/MainMenuManager.cs]: Rename to About, and make it empty (put a TODO comment)

File: Assets/Scripts/MainMenuManager.cs [hunk 4]
```diff
@@ -77,4 +73,9 @@ private int menuSelection(string[] menuItems, int selectedItem, string direction
 
         return selectedItem;
     }
+
+    public void NewGame()
+    {
+        SceneManager.LoadScene("Level_01");
```
Comment (burtybob) [inline Assets/Scripts/MainMenuManager.cs]: There is an ExtendedSceneManager that will take the Enum value for SceneNames now.

File: Assets/Scripts/PlayerController.cs [hunk 5]
```diff
@@ -78,4 +78,14 @@ private bool ShouldJump()
         return Input.GetButton("Jump") && rgBody.velocity.y == 0;
     }
 
+    void OnTriggerEnter2D(Collider2D coll)
+    {
+        if (coll.gameObject.CompareTag("Coin"))
```
Comment (burtybob) [inline Assets/Scripts/PlayerController.cs]: There is an extension method and Tag enum for tags now.

**Pull request merged**